Apps/Src/main.c

```diff
@@ -1,12 +1,39 @@
-/* Copyright (c) 2020 UT Longhorn Racing Solar */
 
-/** main.c
- * Program for UTSVT BeVolt's Battery Protection System
- */
-
-#include "common.h"
+#include "BSP_CAN.h"
 #include "config.h"
+#include "BSP_ADC.h"
+#include <string.h>
+/* Copyright (c) 2021 UT Longhorn Racing Solar */
+
+#define CAN_ID 0x10b //CAN ID
+ 
+int main(void){
+    BSP_ADC_Init();
+    BSP_CAN_Init();
+    volatile uint8_t mailBoxNum;
```

Comment:
There is a lot of debugging code in here. I think it is fine to leave it in for now, since we don't know if this will work when we try to integrate with the display board. After we know this works, we should remove all fo the debugging code to clean it up

Reply:
I rearranged some lines in the variable declarations and while loop , so now, what needs to be removed after integration is clearly marked.